k8s/agones/Makefile

```diff
@@ -0,0 +1,112 @@
+include ../app.Makefile
+include ../crd.Makefile
+include ../gcloud.Makefile
+include ../var.Makefile
+
+TAG ?= 0.8.1
+DEMO_RELEASE_TAG ?= release-0.8.0
+NAME ?= agones
+
+
+$(info ---- TAG = $(TAG))
+$(info ---- DEMO_RELEASE_TAG = $(DEMO_RELEASE_TAG))
+
+
+# source repo and destination tags for images
+SOURCE_REGISTRY ?= gcr.io/agones-images
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/agones/deployer:$(TAG)
+AGONES_CONTROLLER_IMAGE ?= $(REGISTRY)/agones:$(TAG)
+AGONES_SDK_IMAGE ?= $(REGISTRY)/agones/agones-sdk:$(TAG)
+AGONES_PING_IMAGE ?= $(REGISTRY)/agones/agones-ping:$(TAG)
+
+
+XONOTIC_DEMO_BASE=https://github.com/GoogleCloudPlatform/agones/raw/$(DEMO_RELEASE_TAG)/examples/xonotic
+
+
+APP_PARAMETERS ?= { \
+    "APP_INSTANCE_NAME": "$(NAME)", \
+    "NAMESPACE": "$(NAMESPACE)" \
+}
+
+
+APP_TEST_PARAMETERS ?= { \
```

Comment:
If this can be empty, please just remove it.

Reply:
Done.

Reply:
Make variable 'APP_TEST_PARAMETERS' is required
/workspace/k8s//var.Makefile:56: recipe for target '.build/var/APP_TEST_PARAMETERS-required' failed
make: *** [.build/var/APP_TEST_PARAMETERS-required] Error 1
make: *** Waiting for unfinished jobs....